spark/src/main/scala/org/apache/comet/CometSparkSessionExtensions.scala

```diff
@@ -626,6 +635,15 @@ object CometSparkSessionExtensions extends Logging {
     op.isInstanceOf[CometBatchScanExec] || op.isInstanceOf[CometScanExec]
   }
 
+  private def applyRowToColumnar(conf: SQLConf, op: SparkPlan): Boolean = {
```

Comment:
nit: maybe call this `shouldApplyRowToColumnar`. `applyRowToColumnar` sounds like this method will apply the action.